LICENSE

```diff
@@ -0,0 +1,178 @@
+Original Code:
+
+Copyright (c) FatherToast
+
+All Rights Reserved
+
+Special Permission have been granted only to the GTNH Team to decompile and redistribute a derivative version.  See PERMS.png for details.
+
+
+
+GTNH Changes:
```

Comment:
Let's dual license the gtnh changes.  MIT for upstream and lgpl for everyone else.  I tend to do this for any author still active that might want to incorporate any of our changes back into theirs.

Reply:
Do you know a repo I can take as an example?

Reply:
Ftb utils

Reply:
https://github.com/GTNewHorizons/FTB-Utilities/blob/master/README.md


This would need to be in the license here because we aren't licensing the whole mod, just our changes.